SCons/Tool/packaging/__init__.py

```diff
@@ -239,20 +239,19 @@ def options(opts):
 #
 
 def copy_attr(f1, f2):
-    """ copies the special packaging file attributes from f1 to f2.
+    """ Copies the special packaging file attributes from f1 to f2.
     """
     if f1._tags:
         pattrs = [
             tag
             for tag in f1._tags
-            if lambda tag: not hasattr(f2, tag) and tag.startswith('PACKAGING_')
+            if not hasattr(f2, tag) and tag.startswith('PACKAGING_')
         ]
         for attr in pattrs:
             f2.Tag(attr, f1.GetTag(attr))
 
 def putintopackageroot(target, source, env, pkgroot, honor_install_location=1):
-    """ Uses the CopyAs builder to copy all source files to the directory given
-    in pkgroot.
+    """ Copies all source files to the directory given in pkgroot.
```

Comment:
I'd leave the reference to the CopyAs builder here.

Reply:
OK. I figured it wasn't really relevant to the called what the internal mechanism is, but I can put it back.